pilco/environments/__init__.py

```diff
@@ -1 +1 @@
-from .environment import *
+from .environment import Environment
```

Comment:
I think we should be doing `from .environment import *`, and instead of specifying what stuff to import, the appropriate stuff should be declared in the `__all__ = [ ... ]` statement in `environment.py`.